src/components/buttons/SaveButton.tsx

```diff
@@ -10,7 +10,7 @@ import classNames from 'classnames';
 export default function SaveButton(): JSX.Element {
   const { data: isSaving } = useQuery({
     queryKey: ['state', 'isSaving'],
-    placeholderData: false,
+    queryFn: () => false,
```

Comment:
now that we are displaying errors, this was actually showing an error that queryFn was missing